woocommerce-gateway-ebanx/includes/class-wc-ebanx-global-gateway.php

```diff
@@ -74,6 +80,24 @@ public function is_available()
 	}
 
 	/**
+	 * Error handling
+	 *
+	 */
+	public function validate_due_date_days_field() {
+		if ($_POST['woocommerce_ebanx-global_due_date_days'] < 1) {
+			$_POST['woocommerce_ebanx-global_due_date_days'] = self::$defaults['due_date_days'];
+
+			$this->notices
+				->with_message('Days To Expiration must be greater than or equal to 1.', 'woocommerce-gateway-ebanx')
```

Comment:
Take a look at this example: `__('Days to Expiration', 'woocommerce-gateway-ebanx')`